go.mod

```diff
@@ -2,12 +2,53 @@ module github.com/kubeedge/mappers-go
 
 go 1.14
 
+replace (
+	github.com/apache/servicecomb-kie v0.1.0 => github.com/apache/servicecomb-kie v0.0.0-20190905062319-5ee098c8886f
```

Comment:
This vendor can be deleted.

Reply:
Mappers-go import kubeedge device CRD, now kubeedge still need this vendor, cannot remove it.